SPCTR-1891: Fix - Forms phone country code color and select control bgActiveColor not working as intended. (#3271)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###
* Fix: Forms - Resolved an issue in which phone country-code input color would not work as intended on the backend.
* Fix: Forms - Resolved an issue in which phone country-code select-control background active color would not work as intended on the backend.

### 2.6.8 - WEDNESDAY, 21st JUNE 2023 ###
* Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
* Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.

> * Fix: Form - Resolved an issue where the submit button is visible on the frontend when no preset from the variation picker is selected.

Reviewer: @rajmohanreddy774 Please remove the duplicate changelog.

* Fix: Buttons - Resolved an issue in which the space between the icon and text would not apply in RTL sites.
* Fix: Image - Resolved an issue in which the tablet and mobile border css would not be applied as intended.
* Fix: Forms - Reworked the e-mail sending functionality to ensure that only one e-mail is sent when CC and BCC addresses are specified.

> * Fix: Forms - Resolved an issue in which phone country-code input color would not work as intended on the backend.
> * Fix: Forms - Resolved an issue in which phone country-code select-control background active color would not work as intended on the backend.

Reviewer 2: Can we make these two into 1 changelog entry?

```suggestion
* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
```

Reviewer 3: @rajmohanreddy774 pleasse take care of this.
Forwarding to QA

Author: Done.
